.github/workflows/ci.yml

```diff
@@ -51,7 +51,8 @@ jobs:
         # Keep in sync with the version in GitLabDockerContainer.cs
         # Available tags: https://hub.docker.com/r/gitlab/gitlab-ee/tags
         gitlab: [
-          'gitlab/gitlab-ee:14.0.6-ee.0',
+          'gitlab/gitlab-ee:14.2.5-ee.0',
```

Comment:
You need to keep in sync with

https://github.com/ubisoft/NGitLab/blob/4fb258b134e59b9a8e0701e269c449532a060f03/NGitLab.Tests/Docker/GitLabDockerContainer.cs#L33